classes/UpgradeSelfCheck.php

```diff
@@ -278,8 +277,7 @@ private function checkShopIsDeactivated()
     {
         return
             !Configuration::get('PS_SHOP_ENABLE')
-            || (isset($_SERVER['HTTP_HOST']) && in_array($_SERVER['HTTP_HOST'], array('127.0.0.1', 'localhost')))
-        ;
+            || (isset($_SERVER['HTTP_HOST']) && in_array($_SERVER['HTTP_HOST'], array('127.0.0.1', 'localhost')));
```

Comment:
@Quetzacoalt91 Can we add the ipv6 notation here? `[::1]`

Reply:
Yep, we should do that. But I guess in another PR.

Reply:
Created #276